### PR TITLE
(IAC-1408) Release Prep v6.10.0

### PR DESCRIPTION
**Pull request by sheenaajay**

### CODE REVIEW

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
FYI: we did see a pipeline fail on a PostgreSQL service restart which was running Puppet 6. I'm debugging that now, but if that's the same bug as #1230 then perhaps it needs additional work.

**Comment by ekohl** [comment]
In the review of #1230 I was very much under the impression that it only affected Puppet 5. However, in our system upgrades we also see a problem with Puppet 6 which doesn't perform the `daemon-reload`. It is my theory that `/bin/systemctl show --property=NeedDaemonReload -- postgresql` returns something other than `yes` so that https://github.com/puppetlabs/puppet/blob/97ca00bcb935a2efe7a6cac1b8e071418b9a3dc4/lib/puppet/provider/service/systemd.rb#L150-L153 is never triggered. I think that's probably because the unit file is missing entirely according to systemd.

I'll try to write up a PR and I'd appreciate it if that could make it into the release.

**Comment by sheenaajay** [comment]
> In the review of #1230 I was very much under the impression that it only affected Puppet 5. However, in our system upgrades we also see a problem with Puppet 6 which doesn't perform the `daemon-reload`. It is my theory that `/bin/systemctl show --property=NeedDaemonReload -- postgresql` returns something other than `yes` so that https://github.com/puppetlabs/puppet/blob/97ca00bcb935a2efe7a6cac1b8e071418b9a3dc4/lib/puppet/provider/service/systemd.rb#L150-L153 is never triggered. I think that's probably because the unit file is missing entirely according to systemd.
> 
> I'll try to write up a PR and I'd appreciate it if that could make it into the release.

Thanks alot @ekohl  will hold on the release .

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v6.10.0](https://github.com/puppetlabs/puppetlabs-postgresql/tree/v6.10.0) (2021-02-05)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-postgresql/compare/v6.9.0...v6.10.0)
+
+### Added
+
+- Set default PostgresQL version for FreeBSD [\#1227](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1227) ([olevole](https://github.com/olevole))
```
Comment (ekohl) [inline CHANGELOG.md:11]: Nit: should it be fixed to
```suggestion
- Set default PostgreSQL version for FreeBSD [\#1227](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1227) ([olevole](https://github.com/olevole))
```

**Review by ekohl — COMMENTED** [review]
I submitted https://github.com/puppetlabs/puppetlabs-postgresql/pull/1233 and would like to see it included.

**Comment by sheenaajay** [comment]
> I submitted #1233 and would like to see it included.

Yeah sure. Thank you

**Review by ekohl — APPROVED** [review]

**Pull request merged**